composables/use-paywall-modal.ts

```diff
@@ -0,0 +1,121 @@
+import { PaywallModal } from '#components'
+
+export function usePayWall({
```

Comment:
paywall one word
```suggestion
export function usePaywall({
```

I think this should rename `useSubscription`?

Reply:
[🚚 Rename usePaywall to useSubscription](https://github.com/likecoin/3ook-com/pull/57/commits/02a8f18c4ef658afbafd7b3b55a17550a55bfff0)